core/src/main/java/org/mule/routing/Foreach.java

```diff
@@ -191,7 +191,15 @@ public void initialise() throws InitialisationException
             ExpressionConfig expressionConfig = new ExpressionConfig();
             expressionConfig.setExpression(collectionExpression);
             checkEvaluator(expressionConfig);
-            splitter = new ExpressionSplitter(expressionConfig);
+            splitter = new ExpressionSplitter(expressionConfig)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This class has 7 parents which is greater than 5 authorized. ([squid:MaximumInheritanceDepth](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7649&issues=AWi_Zq5sotIXnmsUw8q6&open=AWi_Zq5sotIXnmsUw8q6)

[ISSUE_KEY]: AWi_Zq5sotIXnmsUw8q6